src/emailpro/account/add/emailpro-account-add.controller.js

```diff
@@ -122,7 +129,10 @@ angular.module('Module.emailpro.controllers')
       // $scope.setPasswordsFlag($scope.accountToAdd);
       if (!$scope.valid.legalWarning) {
         return false;
-      } if ($scope.simplePasswordFlag || $scope.differentPasswordFlag || $scope.containsNameFlag) {
+      } if ($scope.simplePasswordFlag
```

Comment:
As it is duplicated, could be grouped to a function something like `isPasswordValid()`

Reply:
it's worse than it looks, the two controllers account-add & account-update share a lots of similarities and duplicated code but one controller uses "$scope", the other on uses "this".

the part of code you highlighted belongs to the controller's logic and cannot be exported in a service (the variables are just flags related to the inputs in the template); to avoid duplication we should completely refactor those two controllers into one but it's out of the scope of this PR :)